data/json/terrain.json

```diff
@@ -3156,9 +3156,29 @@
         "id" : "t_tree_maple",
         "name": "maple tree",
         "symbol": "7",
-        "color": "red",
+        "color": [ "red", "light_green", "light_green", "red" ],
         "move_cost": 0,
-        "flags": ["FLAMMABLE_ASH", "NOITEM", "SUPPORTS_ROOF", "TREE", "REDUCE_SCENT"],
+        "flags": [ "FLAMMABLE_ASH", "NOITEM", "SUPPORTS_ROOF", "TREE", "REDUCE_SCENT" ],
+        "examine_action": "tree_maple",
+        "bash": {
+            "str_min": 80, "str_max": 180,
+            "sound": "crunch!",
+            "sound_fail": "whack!",
+            "ter_set": "t_dirt",
+            "items": [
+                { "item": "stick", "count": [3, 10] },
+                { "item": "splinter", "count": [10, 25] }
+            ]
+        }
+    },{
+        "type" : "terrain",
+        "id" : "t_tree_maple_tapped",
+        "name": "maple tree",
+        "symbol": "7",
+        "bgcolor": "light_cyan",
+        "move_cost": 0,
+        "flags": [ "FLAMMABLE_ASH", "SUPPORTS_ROOF", "TREE", "REDUCE_SCENT" ],
+        "examine_action": "tree_maple_tapped",
```

Comment:
I don't see the need for the generic "tappedness" concept in the game. We do not currently have any other tree type that could be tapped to provide any meaningful product. I believe the only other real tree to be tapped in the real world is the rubber tree to obtain latex. Unless someone is going to implement the full latex/rubber cycle, I don't think it would be worth it.